src/HomePage/Details.js

```diff
@@ -0,0 +1,90 @@
+import { useLocation } from 'react-router-dom';
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Table from 'react-bootstrap/Table';
+import { nanoid } from '@reduxjs/toolkit';
+import { getStats } from '../Redux/stats';
+
+function Details() {
+  const location = useLocation();
+  const { state } = location;
+  const dispatch = useDispatch();
+  const namdr = state.state.driverId;
+  const statsData = useSelector((state) => state.stats);
+  // console.log(statsData);
+  useEffect(() => {
+    if (statsData.length === 0) {
+      dispatch(getStats(namdr));
+    } if (statsData.length > 1) {
+      dispatch(getStats(namdr));
+    }
+    // return () => {
+    //   dispatch(actions.clear());
+    // };
+  });
+  return (
+    <div className="dStandingList">
+      <h1 className="tableTitle">2022 Stats</h1>
+      <Table striped bordered hover size="sm" className="mt-4 mx-auto w-75">
+        <thead>
+          <tr>
+            <th>Circuit</th>
+            <th>Finish</th>
+            <th>Points</th>
+          </tr>
+        </thead>
+        <tbody>
+          {statsData.map(
+            (stat) => (
+              // eslint-disable-next-line no-sequences
+              // console.log(stat),
```

Comment:
- [ ] Please do not disable `eslint` it will help you to write a high quality code, instead you can challenge yourself and try to fix the issue 💪🏻

Reply:
well resolved! ✔️